src/WingetCreateCLI/Properties/Resources.resx

```diff
@@ -658,4 +649,19 @@
   <data name="InstallerCacheCleaned_Message" xml:space="preserve">
     <value>Installer cache cleaned.</value>
   </data>
+  <data name="DeletingInvalidCachedToken_Message" xml:space="preserve">
+    <value>Cached token was invalid, deleting token from cache...</value>
+  </data>
+  <data name="InvalidGitHubToken_Message" xml:space="preserve">
+    <value>Token was invalid. Please generate a new GitHub token and try again.</value>
+  </data>
+  <data name="RateLimitExceeded_Message" xml:space="preserve">
+    <value>GitHub api rate limit exceeded. To extend your rate limit, provide your GitHub token with the "-t" flag or store one using the "token --store" command.</value>

```

Comment:
I'm not sure if the localizers require you to explicitly state that parameters like this shouldn't be localized?

Reply:
Added explicit comment explaining that these are parameters